packages/client/src/components/Auth/Signin.jsx

```diff
@@ -55,6 +63,20 @@ export const Signin = () => {
           <Link to="/signup"> Signup</Link>
         </Typography>
       </Form>
+
+      <Modal open={isModalOpen}>
+        {error ? (
+          <ModalBox backgroundColor="hoverRed">
+            <Typography variant="h5">
+              Something went wrong, please try again.
```

Comment:
"The email address or password you entered is invalid" for the appropriate cases and the current message for everything else would be better UX 🙏🏽

Reply:
Fixed!